app/controllers/api/user_blocks_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  class UserBlocksController < ApiController
+    before_action :check_api_readable
+
+    authorize_resource
+
+    around_action :api_call_handle_error, :api_call_timeout
+    before_action :set_request_formats
+
+    def show
+      raise OSM::APIBadUserInput, "No id was given" unless params[:id]
```

Comment:
Maybe make it a bit clearer, which "id" we expect here, like "No user block id was given".